eisbuk-admin/src/pages/attendance_printable/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import Container from "@material-ui/core/Container";
+
+import AppbarAdmin from "@/components/layout/AppbarAdmin";
+import DateNavigation from "@/components/atoms/DateNavigation";
+import AttendanceSheet from "@/components/atoms/AttendanceSheet";
+import { AttendanceSheetSlot } from "@/components/atoms/AttendanceSheet";
```

Comment:
There should only be one import of one module (`import AttendanceSheet, {AttendanceSheetSlot} from "@/components/atoms/AttendanceSheet"`)